playwright-e2e/dsm/pages/participant-page/rgp/rgp-participant-page.ts

```diff
@@ -0,0 +1,91 @@
+import ParticipantPage from '../participant-page';
+import { Locator, Page } from '@playwright/test';
+
+/**
+ * Captures the webelements in between profile section and the Family Member section
+ */
+export default class RgpParticipantPage extends ParticipantPage {
+    constructor(page: Page) {
+        super(page);
+    }
+
+    public getAddFamilyMemberButton(): Locator {
+        return this.page.locator("//span[contains(text(), 'Add Family Member')]/preceding-sibling::button");
+    }
+
+    public getAddFamilyMemberPopup(): Locator {
+        return this.page.locator("//mat-dialog-container//mat-dialog-content//div[@class='family-member-popup']");
+    }
+
+    public getFamilyMemberFirstName(): Locator {
```

Comment:
Recommend you do something like `addFamilyMemberDialog()` in my draft [PR](https://github.com/broadinstitute/ddp-angular/pull/2132/files).